src/main/java/com/spotify/docker/BuildMojo.java

```diff
@@ -601,6 +664,30 @@ private void createDockerFile(final String directory, final List<String> filesTo
     return allCopiedPaths;
   }
 
+  /**
+   * When running on windows, paths are backslash separated,
+   * we need to convert them to unix '/' separated paths.
+   *
+   * @param path
+   *            object (system specific)
+   * @return unix style path
+   */
+  private String unixPath(Path path) {
+    StringBuilder sb = new StringBuilder();
```

Comment:
Maybe I'm missing something, but why can't we just replace all backslashes with slashes and call it a day? That's [what Apache Commons IO seems to do](http://commons.apache.org/proper/commons-io/apidocs/src-html/org/apache/commons/io/FilenameUtils.html#line.553).